Apps/Src/Amps.c

```diff
@@ -84,11 +84,11 @@ ErrorStatus Amps_UpdateMeasurements(void) {
 
 /** Amps_CheckStatus
  * Checks if pack does not have a short circuit
- * @param chargingOnly a flag that indicates whether we should allow
- * 		  only charging or both charging and discharging.
+ * @param maxTemperature the maximum temperature measured in the most recent batch of temperature readings
  * @return SAFE or DANGER
  */
-SafetyStatus Amps_CheckStatus(bool chargingOnly) {
+SafetyStatus Amps_CheckStatus(int32_t maxTemperature) {
+	bool chargingOnly = maxTemperature > MAX_CHARGE_TEMPERATURE_LIMIT;
```

Comment:
Would this mean we want to only charge when the max temperature is larger than the MAX_CHARGE_TEMPERATURE_LIMIT? Shouldn't it be the other way around, only discharge if it is greater than this limit (since the charge limit is 45C and the discharge limit is 60C)?

Reply:
Yeah, you're right. I'll fix this